README.md

```diff
@@ -69,6 +69,25 @@ npm login --scope=@block-wallet --registry=https://npm.pkg.github.com
 
 That command will ask for your GitHub username, password (use the GitHub token generated before) and also the email.
 
+#### Environment
+
+It's also needed to setup the environment variables on each package. In both ``~/packages/background`` and ``~/packages/provider``, copy the contents of ``env.orig`` file into a new file called ``.env``.
+
+They should look like:
+
+**packages/background/.env**
+```
+// Complete with the corresponding keys.
+INFURA_PROJECT_ID=
```

Comment:
should we add a section with the explanation of these 2 secrets or at least the urls?

Reply:
done!